optimize/util/optimize-commons/src/main/java/io/camunda/optimize/dto/optimize/index/PositionBasedImportIndexDto.java

```diff
@@ -52,39 +52,41 @@ public void setHasSeenSequenceField(final boolean hasSeenSequenceField) {
   }
 
   @Override
-  public String toString() {
-    return "PositionBasedImportIndexDto(positionOfLastEntity="
-        + getPositionOfLastEntity()
-        + ", sequenceOfLastEntity="
-        + getSequenceOfLastEntity()
-        + ", esTypeIndexRefersTo="
-        + getEsTypeIndexRefersTo()
-        + ", hasSeenSequenceField="
-        + isHasSeenSequenceField()
-        + ")";
+  protected boolean canEqual(final Object other) {
+    return other instanceof PositionBasedImportIndexDto;
   }
 
   @Override
-  public boolean equals(final Object o) {
-    return org.apache.commons.lang3.builder.EqualsBuilder.reflectionEquals(this, o);
+  public int hashCode() {
+    return org.apache.commons.lang3.builder.HashCodeBuilder.reflectionHashCode(this);
   }
 
   @Override
-  protected boolean canEqual(final Object other) {
-    return other instanceof PositionBasedImportIndexDto;
+  public boolean equals(final Object o) {
+    return org.apache.commons.lang3.builder.EqualsBuilder.reflectionEquals(this, o);
   }
 
   @Override
-  public int hashCode() {
-    return org.apache.commons.lang3.builder.HashCodeBuilder.reflectionHashCode(this);
+  public String toString() {
+    return "PositionBasedImportIndexDto(positionOfLastEntity="
+        + getPositionOfLastEntity()
+        + ", sequenceOfLastEntity="
+        + getSequenceOfLastEntity()
+        + ", dbTypeIndexRefersTo="
+        + getDbTypeIndexRefersTo()
+        + ", hasSeenSequenceField="
+        + isHasSeenSequenceField()
+        + ")";
   }
 
   @SuppressWarnings("checkstyle:ConstantName")
   public static final class Fields {
 
     public static final String positionOfLastEntity = "positionOfLastEntity";
     public static final String sequenceOfLastEntity = "sequenceOfLastEntity";
-    public static final String esTypeIndexRefersTo = "esTypeIndexRefersTo";
+    // With the support of Opensearch it was decided to get rid of ES prefixes in variable names.
+    // However, to avoid reindexing it was decided to keep original field names in indices
+    public static final String dbTypeIndexRefersTo = "esTypeIndexRefersTo";
```

Comment:
Is there a reason why we didn't promote this field to the ImportIndexDto like in C7?

Reply:
I guess the reason could be that this field in theory might be different for those entities, but as long as it is same it can be put in ImportIndexDto. Changed it